### DISCUSSION TITLE
Adding a Budgie Edition to Mint, Possibly LMDE

### DISCUSSION THREAD

**Discussion by WesfunOfficial** [linuxmint/discussions#465] [Ideas]
Me and a few people outside of the discussions area have been playing around with the idea of if budgie where to be an official edition of mint/lmde. if a repo could be created in the same way xfce and mate, i would be more than happy to contribute personally

**Comment by Qronikarz** [comment]
I think it will be impossible.

Budgie has long ago declared to leave GTK world due to issues with GTK4 (few years before Linux Mint developers) and Budgie 11 won't use GTK.

Mint developers have said that they want to focus only on GTK DEs (because then it's less work since 3 editions can have the same base or something like that). It's the most common answer that people bring up when lost souls ask for Mint KDE edition.
